source/test-suite/Main.hs

```diff
@@ -32,31 +32,31 @@ main = Hspec.hspec . Hspec.parallel . Hspec.describe "Imp" $ do
     expectImp
       []
       "true = Data.Bool.True"
-      "import qualified Data.Bool\ntrue = Data.Bool.True"
+      "import (implicit) qualified Data.Bool\ntrue = Data.Bool.True"
```

Comment:
I didn't realize that implicit imports are output differently than normal ones. It's a bit weird that they're not even valid syntax, but I suppose that makes sense because they're supposed to be implicit anyway. It also makes the tests a little nicer because I can be sure the imports were actually inserted by Imp (rather than somehow coming from the input or parser).